lang/src/org/partiql/lang/eval/builtins/MathFunctions.kt

```diff
@@ -0,0 +1,46 @@
+package org.partiql.lang.eval.builtins
+
+import org.partiql.lang.eval.EvaluationSession
+import org.partiql.lang.eval.ExprFunction
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueFactory
+import org.partiql.lang.eval.numberValue
+import org.partiql.lang.types.AnyOfType
+import org.partiql.lang.types.FunctionSignature
+import org.partiql.lang.types.StaticType
+import kotlin.math.ceil
+import kotlin.math.floor
+
+/**
+ * A place to keep supported mathematical functions. We are missing many in comparison to PostgresQL.
+ * https://www.postgresql.org/docs/9.5/functions-math.html
+ */
+object MathFunctions {
+
+    fun create(valueFactory: ExprValueFactory): List<ExprFunction> = listOf(
+        UnaryNumericToInt("ceil", valueFactory) { ceil(it.toDouble()) },
+        UnaryNumericToInt("ceiling", valueFactory) { ceil(it.toDouble()) },
+        UnaryNumericToInt("floor", valueFactory) { floor(it.toDouble()) },
+    )
+}
+
+private val numericTypes =
+    setOf(StaticType.INT2, StaticType.INT4, StaticType.INT8, StaticType.INT, StaticType.FLOAT, StaticType.DECIMAL)
+
+class UnaryNumericToInt(
```

Comment:
Can some documentation (i.e. [kdoc](https://kotlinlang.org/docs/kotlin-doc.html)) be added for this function? 

Also consider making this `private` or `internal`